docs/src/main/asciidoc/kogito-dev-services-build-time-config.adoc

```diff
@@ -0,0 +1,54 @@
+[.configuration-legend]
```

Comment:
Isn't this file auto-generated?

Reply:
It is but in this case it comes from an outside extension.